azurerm_private_endpoint/main.tf

```diff
@@ -21,3 +21,18 @@ resource "azurerm_private_endpoint" "private_endpoint" {
   }
 
 }
+
+
+module "private_dns_zone" {
```

Comment:
So you are going to create a new private_dns_zone for each private_endpoint?
Why you think is needed?

Reply:
No it's optional:
module_disabled = var.private_dns_zone_name == null ? true : false
It's basically what the portal lets you do: when you crate the private endpoint you can also create the dns zone.